package.json

```diff
@@ -12,7 +12,9 @@
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.14.1"
+    "react-hook-form": "^7.45.2",
+    "react-router-dom": "^6.14.1",
+    "yup": "^1.2.0"
```

Comment:
@WONILLISM What is yup?!

Reply:
Yup is a JavaScript library that provides a simple and easy way to perform schema validation for objects, ensuring that they meet specific criteria or rules.